WordPress/Classes/Services/EditorSettingsService.swift

```diff
@@ -5,8 +5,26 @@ import Foundation
 }
 
 @objc class EditorSettingsService: LocalCoreDataService {
-    @objc func syncEditorSettings(for blog: Blog, success: @escaping () -> Void, failure: @escaping (Swift.Error) -> Void) {
-        guard let api = blog.wordPressComRestApi(), let siteID = blog.dotComID?.intValue else {
+
+    let database: KeyValueDatabase
```

Comment:
Temporary addition to be able to inject a Mock KeyValueDatabase back to GutenbergSettings on Unit Tests.
This will be removed on https://github.com/wordpress-mobile/WordPress-iOS/pull/12196